codec-dns/pom.xml

```diff
@@ -76,6 +76,12 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec-http2</artifactId>
```

Comment:
I dont think dns-codec module is expected to have http2 related compile dependencies; there is marginal demand for dns-over-https, and http related dependencies are exported to downstream projects which do not necessary need them. How about separate module?

Reply:
Thanks for your response. 
I'll create a separate module to use the netty-codec-http2 dependency without causing any further issues.